src/api/app/views/webui2/webui/staging/workflows/new.html.haml

```diff
@@ -4,21 +4,23 @@
   = render(partial: 'webui2/webui/project/tabs', locals: { project: @project })
   .card-body
     %h3= @pagetitle
-    .mb-3
+    %p
       Adding the staging capability to this project will enable you to group
       requests into staging projects and being able to see how they build together.
+    %p
+      Managers groups can only be created by an admin for now.
```

Comment:
I would drop the 'for now'. It could be that this stays there forever and if it changes some day we would change this message.

And I would direct user towards a solution here, like 'Please contact an admin if don't have a group yet'.

Reply:
Good suggestion. In addition to this, I would like to direct the user even more. What is the best way to contact an admin? IRC, our mailing list, something else?

Reply:
Well, it depends on the instance. For OBS it's an email address, admin@opensuse.org. That one is stored in [Configuration[:admin_email]](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/configuration.rb#L36)